csvcubed/csvcubed/cli/inspect/metadataprocessor.py

```diff
@@ -0,0 +1,33 @@
+"""
+Metadata Processor
+---------------------------
+
+Provides functionality for validating and detecting input metadata.json file.
+"""
+
+import logging
+
+from pathlib import Path
+from csvcubed.cli.inspect.metadatainputhandler import MetadataType
+from rdflib import Graph
+
+_logger = logging.getLogger(__name__)
+
+
+class MetadataProcessor:
+    """
+    This class handles the conversion/serialisation of metadata jsons to RDF.
+    """
+
+    def serialise_to_rdf(self, metadata_json: Path, type: MetadataType) -> Graph:
+        """
+        Serialises given metadata json to RDF.
+
+        Member of :class:`./MetadataProcessor`.
+
+        :return: `Graph` - RDF Graph of CSV-W metadata.
+        """
+
+        """TODO: Convert Metadata json to RDF"""
+        graph = Graph()
```

Comment:
[This](https://github.com/GSS-Cogs/csvcubed/blob/a9382c7b1ed748c786dd722865e411d17d21ecc4/csvcubed-pmd/csvcubedpmd/dcatcli/pmdify.py#L48) provides an example of how to read a JSON-LD file with `rdflib.Graph`.